source/developers-guide/csrf-protection/index.md

```diff
@@ -107,3 +107,25 @@ In some cases you might want to disable the protection for the backend or fronte
 ],
 ...
 ```
+
+## Plugin compatibility for older versions
+
+Since Shopware 5.2, your plugin needs to whitelist an action in order to e.g. transfer files or show a page within an window or iframe. For Shopware versions prior to 5.2, the interface `CSRFWhitelistAware` isn't available and you'll receive an exception. In this case, you have to create a dummy interface which will only be loaded if it does not exists.
```

Comment:
In that case, you have to create a dummy interface which will only be loaded if the original one does not exists.